### PR TITLE
Added steps to get config_entry for OpenAI Conversation example

### LINKED ISSUE #28615: The sample automation from the OpenAI Conversation integration doesn't work
### Feedback

IMG_5060

IMG_5059

### URL

https://www.home-assistant.io/integrations/openai_conversation/

### Version

2023.8.2

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by dannytsang**
## Proposed change
config_entry for OpenAI Conversation seems to be installation dependent so I have added steps to get this using developer tools. See #28615 for more details.

Fix template syntax for **state** to **states** of weather.home.

Added quotes to size parameter.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #28615

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/openai_conversation.markdown [hunk 1]
```diff
@@ -87,6 +87,15 @@ to generate a new image of New York in the current weather state.
 
 The resulting image entity can be used in, for example, a card on your dashboard.
 
+The *config_entry* is installation specific. To get the value, make sure the integration has been installed.
+The go to {% my developer_services title="**Developer Tools** > **Services**" %}. Ensure you are in UI mode and enter the following below:
```
Comment (c0ffeeca7) [inline source/_integrations/openai_conversation.markdown]:
```suggestion
Then, go to {% my developer_services title="**Developer Tools** > **Services**" %}. Ensure you are in UI mode and enter the following below:
```

**Review by c0ffeeca7 — APPROVED** [review]
Nice :star_struck:  Thank you, @dannytsang

**Comment by frenck** [comment]
@dannytsang @c0ffeeca7 isn't this a generic thing we should solve centrally/using includes to prevent we do/add this dozen of times in all places that use a config entry ID?

Ps: normally a user shouldn't need this, this is only needed if a user manually writes automations. I think it should therefore be collapsed as well.

File: source/_integrations/openai_conversation.markdown [hunk 2]
```diff
@@ -99,10 +108,10 @@ automation:
         service: openai_conversation.generate_image
         response_variable: generated_image
         data:
-          config_entry: f29e6b8696a15e107b4bd843de722249
-          size: 512
+          config_entry: abce6b8696a15e107b4bd843de722249
```
Comment (balloob) [inline source/_integrations/openai_conversation.markdown:111]: We should not include a config entry ID here at all. Just some text to "<INSERT OWN CONFIG ENTRY ID>"

**Pull request merged**